.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Issue Template
+about: File an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+This is a suggested issue template for tedana.
+
+If there is other information that would be helpful to include, please don not hesitate to add it!
+
+Before submitting, please check to make sure that the issue is not already addressed.
+If this is a usage question, please checkout NeuroStars here:
+https://neurostars.org/
+and tag your topic with "multi-echo" so that we can see it.
+-->
+
+<!--
+Summarize the issue in 1-2 sentences, linking other issues if they are relevant
+
+Note: simply typing # will prompt you for open issues to select from
+-->
+### Summary
+
+<!--
+If needed, add additional detail for:
+1. Recreating a bug/problem
+2. Context
+-->
+### Additional Detail
+
+<!--
+If desired, add suggested next steps.
+If you foresee them in a particular order or priority, please use numbering
+-->
+### Next Steps
+
+<!-- 
+Thank you for submitting your issue!
```

Comment:
General question to the group -- should we establish some sort of timeline ? Like, if you don't hear back after X days, please ping again ?

Reply:
I think that would be good. For me personally, I would say one calendar week. But I'm not the one likely to be tagged/notified 😄

Reply:
I think if we're clear that it won't necessarily be _dealt_ with in one week, just acknowledged, then that seems very reasonable to me. WDYT @tsalo @KirstieJane ?

Reply:
LGTM. There's just one typo to fix and then I think it's ready to merge, pending @KirstieJane's input.

Reply:
@tsalo @emdupre @KirstieJane please see if the wording I have in 09da2c7e5803c528b8727fc9abea932d8ee3ed07 addresses the timeline in the way you'd like.